examples/app/stocks/download_sample_data.py

```diff
@@ -26,7 +26,7 @@ def extract_hosted_zip(data_url, save_dir, exclude_term=None):
 def extract_zip(zip_name, exclude_term=None):
     """Extracts a zip file to its containing directory."""
 
-    zip_dir = os.path.dirname(os.path.abspath(zip_name))
+    zip_dir = os.path.dirname(os.path.abspath(os.path.expanduser(zip_name)))
```

Comment:
The download path is path is controlled and always relative to `cwd` this is not needed